scripts/protopackage.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/sh
+set -eo pipefail
+
+# This script appends the "option go_package" proto option to the file located at $FNAME.
```

Comment:
is this idempotent?

Reply:
fixed!

Reply:
To elaborate, we download the proto files, compile them into .go files and then delete the downloaded proto files. Idempotency on the downloaded files is not really that important since we also delete them in the script that calls this one. However, there are some protos that we don't download that already set the option, so we need idemotency for those.

Reply:
```suggestion
# This script appends the "option go_package" proto option to the file located at $FNAME.
# If the file already contains a go_package option, it is replaced.
```